### PR TITLE
feat(price-feed): added USDXIO price feed in DefaultPriceFeedConfigs.js

### PR DESCRIPTION
**Pull request by asadnaeem1**
**Motivation**

Default price feed config for USDXIO price identifier, [UIMP-71](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-71.md).

**Summary**

Default config for the USDXIO price identifier is created in the file DefaultPriceFeedConfigs.js.

**Details**

The config is of type expression. It uses uniswap pairs of XIO/ETH and ETH/USDT to calculate the price of USD in terms of XIO.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [x]  Untested

### CODE REVIEW

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js [hunk 1]
```diff
@@ -938,6 +938,25 @@ const defaultConfigs = {
     uniswapAddress: "0x360acfeb5c1548bad3583c559a646d803077236d",
     twapLength: 7200,
     invertPrice: false
+  },
+  USDXIO: {
```
Comment (smb2796) [inline packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:942]: Alternative configuration that follows the specs in the UMIP. I've tested and this looks to return the expected prices, but please test on your end!

```
  USDXIO: {
    type: "expression",
    expression: `
      USDETH * ETHXIO
    `,
    minTimeBetweenUpdates: 60,
    lookback: 7200,
    customFeeds: {
      ETHXIO: {
        type: "uniswap",
        uniswapAddress: "0xe0cc5afc0ff2c76183416fb8d1a29f6799fb2cdf",
        invertPrice: true,
        twapLength: 3600
      }
    }
  }
```

**Review by smb2796 — COMMENTED** [review]
Hi @asadnaeem1! Thanks for submitting!

The price config does not currently follow the specs of UMIP-71. Specifically the USDETH methodology does not match what is expected. 

I've left a suggestion for a new pf config. Please take a look and test on your end to make sure the resultant prices match your expectations.

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js (hunk 1 above)
Comment (asadnaeem1) [reply packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:942]: Thank you @smb2796, your suggested config is working fine. I have created another commit.

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js [hunk 2]
```diff
@@ -938,6 +938,20 @@ const defaultConfigs = {
...
+    type: "expression",
+    expression: "XIOETH * USDETH",
```
Comment (smb2796) [inline packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:944]:
```suggestion
    expression: "ETHXIO * USDETH",
```

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js [hunk 3]
```diff
@@ -938,6 +938,20 @@ const defaultConfigs = {
...
+    lookback: 7200,
+    minTimeBetweenUpdates: 60,
+    twapLength: 3600,
+    customFeeds: {
+      XIOETH: {
```
Comment (smb2796) [inline packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:949]: This is actually the ETHXIO price.

```suggestion
      ETHXIO: {
```

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js [hunk 4]
```diff
@@ -938,6 +938,20 @@ const defaultConfigs = {
     uniswapAddress: "0x360acfeb5c1548bad3583c559a646d803077236d",
     twapLength: 7200,
     invertPrice: false
+  },
+  USDXIO: {
+    type: "expression",
+    expression: "XIOETH * USDETH",
+    lookback: 7200,
+    minTimeBetweenUpdates: 60,
+    twapLength: 3600,
```
[the remark this answers is not in the document]
Comment (asadnaeem1) [reply packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js]: @smb2796 sure, please update the UMIP. I have moved the twapLength into the custom feed as you recommended.

[the remark this answers is not in the document]
Comment (smb2796) [reply packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js]: Sorry, I should've been clearer here.

Option 1: leave twapLength outside of the customFeed. USDETH and ETHXIO are calculated with a TWAP. This is not in the UMIP, but can be with an update.
Option 2: Move twapLength inside the ETHXIO customFeed to agree with the UMIP.

My recommendation was to go with Option 1 (leave it outside the customFeed) - but just wanted to get your confirmation before updating the UMIP.

[the remark this answers is not in the document]
Comment (asadnaeem1) [reply packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:947]: @smb2796, I have reverted the change

**Comment by smb2796** [comment]
There were some minor CI issues with this PR, so I added the same PF config into #2975. Closing bc of this.

**Pull request closed without merging**